misk/web/modules/config/.babelrc.old

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "travix"
```

Comment:
what is travix?

Reply:
Company out of which `FrintJS` framework came. Can see if their preset can be removed. Have been removing their related pieces when I find them.